pkg/manager/manager_test.go

```diff
@@ -0,0 +1,17 @@
+package manager
```

Comment:
add bcc build tag to avoid getting tested on non-bcc platform

Reply:
see https://github.com/sustainable-computing-io/kepler/pull/441/files#diff-d765c874d837c9e7b84161e25b51a9be7854289d92c4113f5dc96c76bb017d8fR1

Reply:
https://github.com/sustainable-computing-io/kepler/blob/main/Makefile#L51
here is the rc, as bcc tag will added only for ? I suppose centos yum? but for github action, os is ubuntu.

Reply:
the tag is used by `go test` here https://github.com/sustainable-computing-io/kepler/blob/main/Makefile#L145